### ISSUE TITLE
[Feature] add argument for default resolution strategy

### ISSUE THREAD

**Issue by Eidenz** [Sanster/lama-cleaner#43]
I made my lama-cleaner installation available to some people, but since it's hosted on a low-power VPS the only option that works is using `Crop` as the resolution strategy (others end up crashing due to RAM or CPU).

However, with the latest version `Resize` is set as default and I didn't find an argument to switch to `Crop` at startup. Would it be possible to add an argument to change the default selection (and perhaps even lock it)?

**Comment by Sanster** [comment]
This tool is positioned as a personal tool, and I think it makes sense to add a default configuration to the startup options, but I don't plan to support the locking feature.

Settings are saved in js local storage, for now, you can tell your users to choose `Crop` for the first time by themselves.

**Comment by Sanster** [comment]
I want to use localStorage to keep the user's changes of settings, but I can't find a way that is compatible with fetching default settings from the server. How do I decide to use the configuration in localStorage or the default settings from the server?

**Comment by Eidenz** [comment]
I'm not sure to understand the problem...

If there is nothing in localStorage, use server settings. If there is something in localStorage, it means the user changed something and it got saved, so use this instead?

**Comment by Sanster** [comment]
If you restart the service and change the default parameters, but localStorage already exists, the default parameters will not take effect, which will confuse users.

**Comment by Eidenz** [comment]
True. But I think if the user manually changed the parameters, it mostly means they want it that way even if the server default changes.

As a user, I wouldn't like it if my settings reverted back to default just because the server changed default options.


But you could use a timestamp or something to compare the two if you really want to take changes into account?
